lib/component.js

```diff
@@ -68,11 +69,29 @@ function build ( string, opts, readFile ) {
 						promises.push( readFile( src ).then( str => partials[id] = str ) );
 					}
 				} else if ( !type || type === 'text/javascript' || type === 'application/javascript' ) {
+					const rel = getAttr( 'rel', item);
```

Comment:
`<script rel>`?

- Preferring we keep the template at 3 major sections. Otherwise, it gets confusing.
- Taking rcu as reference implementation, [it enforces the single-`<script>`](https://github.com/ractivejs/rcu/blob/master/src/parse.js#L46) logic.
- `<script>` for styles is strange.
- Text editor highlighting will yell at you. 😄 

How about `<style type="text/ractive-fn-css">`? [Ractiveify](https://github.com/norcalli/ractiveify) uses `type` to hint pre-processing. We could use this to hint the bin for special treatment of CSS while being within the gray area of the spec - where there is no defined behavior if we provided different `type`. 😄